docs/src/components/ReactExample/components/knobs/Boolean.tsx

```diff
@@ -6,18 +6,16 @@ import { Knob } from "../../Example";
 
 interface Props extends Pick<Knob, "name"> {
   checked: boolean;
-  onChange: (ev: React.SyntheticEvent<HTMLInputElement, Event>) => void;
+  onChange: (ev: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Sorry, I meant `*EventHandler`, not `*Event` 🤦‍♂️ 🙏
```suggestion
  onChange: React.ChangeEventHandler<HTMLInputElement>;
```